nextflow.config

```diff
@@ -9,13 +9,53 @@
 // Global default params, used in configs
 params {
 
-    // TODO nf-core: Specify your pipeline's command line flags
     // Input options
     input                      = null
-    // References
-    genome                     = null
-    igenomes_base              = 's3://ngi-igenomes/igenomes/'
-    igenomes_ignore            = false
+    transcript_fasta           = null
+
+    // Workflow options
+    QC_only                    = false
+    skip_assembly              = false
+
+    // Trimming options
+    adapter_fasta              = []
+    save_trimmed_fail          = false
+    save_merged                = false
+    skip_fastp                 = false
+    skip_fastqc                = false
+    extra_fastp_args           = null
+
+    // SortMeRNA options
+    remove_ribo_rna            = false
+    save_non_ribo_reads        = false
+    ribo_database_manifest     = "${projectDir}/assets/rrna-db-defaults.txt"
+    sortmerna_index            = []
+
+    // Assembly options
+    trinity                    = true
+    trinity_no_norm            = false
+    extra_trinity_args         = null // applies to both trinity and trinity_no_norm
+    rnaspades                  = true
+    soft_filtered_transcripts  = false
+    hard_filtered_transcripts  = false
+    ss                         = null
+    extra_tr2aacds_args        = null
+
+    // BUSCO options
+    busco_mode                 = 'transcriptome'
+    busco_lineage              = 'auto'
+    busco_lineages_path        = []
+    busco_config               = []
+
+    // rnaQUAST options
+    fasta                      = null
+    gtf                        = null
+
+    // Transrate options
+    transrate_reference        = []
+
```

Comment:
All the parameters here taking an optional file should have null as the default instead of empty lists.
If you need to pass an empty list into modules/subworkflows you can use eg: `params.busco_lineages_path ?: []`